src/components/LogoutButton/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
+import { useDispatch } from 'react-redux';
+import logo from '../../assets/images/dblogo.png';
```

Comment:
Please, add aliases

Reply:
set basicUrl as ./src , so we can write 'assets/images/dblogo.png' now.
I consider it inappropriate to put a third-party library in order to reduce the path by one word.